src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java

```diff
@@ -51,6 +52,18 @@ public EventDtoList getIncompleteEvents() {
         return evList;
     }
 
+    @GET
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/checkDay")
+    public boolean getDay() {
+        User currentUser = userStore.getCurrentUser();
+        Optional<DailyVote> day = voteStore.getCurrentVoteDate(currentUser, LocalDate.now());
+        if(day.isPresent()) {
+            return true;
+        }
+        return false;
```

Comment:
This can be simplified to `return day.isPresent();`

Reply:
Simplified